src/Grid/Source/Vector.php

```diff
@@ -223,7 +223,14 @@ public function getTotalCount($maxResults = null)
 
     public function getHash()
     {
-        return __CLASS__ . md5(implode('', array_map(function ($c) { return $c->getId(); }, $this->columns)));
+        return __CLASS__ . md5(
```

Comment:
If you use an arrow function, you can probably keep this in one line nicely

Reply:
Updated